src/BlazorWebView/src/Maui/BlazorWebView.cs

```diff
@@ -16,5 +17,11 @@ public BlazorWebView()
 		public string? HostPage { get; set; }
 
 		public RootComponentsCollection RootComponents { get; }
+
+		/// <inheritdoc/>
+		public virtual IFileProvider? CreateFileProvider(string contentRootDir)
```

Comment:
Should this maybe be a property? I feel setting `webview.FileProvider = Xxx` is easier than overriding. Unless I am missing some reason why this is better? Maybe even a bindable property so it can be swapped out? Unless swapping is not possible, but then I could swap out via the override logic, so that needs to be supported either way.

Reply:
And, if it is a property, then the handler can have a mapper that can update the native things.

Reply:
Looking at the samples with the overrides... I feel that this is not very XAML-y - an override just to return a single value...

Reply:
And I see your last point on the last comment is not this... Hmmm.

Reply:
I'd love a property if I had a way of passing in a parameter to it... which would mean I need a new interface with such a method on it... which would mean I'd need 3 because of WPF/WinForms/MAUI.